specs/Standards/ContractMetadata.md

```diff
@@ -0,0 +1,43 @@
+# Contract Metadata
+
+## [NEP-330](https://github.com/near/NEPs/discussions/329)
+
+Version `1.0.0`
+
+## Summary
+
+The contract metadata is a standard interface that allows auditing and viewing source code for a deployed smart contract. Implementation of this standard is purely optional but is recommended for developers whose contracts are open source.
+
+## Motivation
+
+There is no trivial way of finding the source code or author of a deployed smart contract. Having a standard that outlines how to view the source code of an arbitrary smart contract creates an environment of openness and collaboration.
```

Comment:
One of the main use cases of having this standard, is if we are able to reproduce the same binary after we have access to the source code. I'm wondering if as part of it we should add a proposal or advise about how we can achieve byte for byte reproducibility. 

Having a standard for reproducibility can be useful for external tools to automatically verify the code.

This is hard in general, mostly because there is more than one stack that can be used to generate the contract binary (i.e today we support rust and assembly script). Relevant discussion on how to make this possible on rust: https://github.com/rust-lang/cargo/issues/9506

Reply:
A link to a docker image could be sufficient.